lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -351,10 +352,26 @@ def extract_details(exam_attempt, features):
                 u'{status} Comments'.format(status=status): '; '.join(comment_list),
             })
 
+        proctored_exam['track'] = course_enrollment[exam_attempt['user_id']]
```

Comment:
if `course_enrollment` is None, it'll raise an exception. not sure if that's possible.